src/Resolver/Value/ValidatedValue.php

```diff
@@ -48,8 +48,47 @@ public function jsonSerialize()
         return $this->value;
     }
 
-    public function printValue() : string
+    public function printValue(bool $isList = false) : string
```

Comment:
If you need new parameter, it is much better to introduce private function and do not modify interface of this function.